jest.config.js

```diff
@@ -0,0 +1,15 @@
+export default {
+  moduleFileExtensions: ['js', 'jsx', 'json', 'vue'],
+  transform: {
+    '^.+\\.(js|jsx)$': 'babel-jest',
+  },
+  moduleNameMapper: {
+    '^@/(.*)$': '<rootDir>/src/$1',
+    '\\.(css|less|sass|scss)$': 'identity-obj-proxy',
+    '\\.(jpg|jpeg|png|gif|svg)$': '<rootDir>/src/tests/fileMock.js', // Add this line to handle image imports
+  },
+  testMatch: ['**/tests/**/*.test.(js|jsx)'],
+  testPathIgnorePatterns: ['/node_modules/', '/public/'],
+  collectCoverageFrom: ['src/**/*.{js,jsx}'],
+  testEnvironment: 'jsdom',
+};
```

Comment:
- [ ] This file is causing the test from running successfully. Kindly fix the error it's giving so the tests can run successfully 👍🏽

<img width="1278" alt="Screenshot 2023-05-19 at 8 28 36 AM" src="https://github.com/Atifzada/Space_Travelers_Hub/assets/50941074/8317c57d-cf3b-412e-a80d-d0eddf23ae0d">

Reply:
Sorry, I forget to delete this file. Thank you for the helpful review